### ISSUE TITLE
ocr_plate_all_gru.h5 模型加载问题

### ISSUE THREAD

**Issue by all150-github** [szad670401/HyperLPR#269]
您好 请问我应该怎么加载 ocr_plate_all_gru.h5 这个模型呢 非常感谢您回答这个问题

**Comment by all150-github** [comment]
我知道 使用 load_model可以加载模型

**Comment by all150-github** [comment]
我知道可以使用 load_weights和load_model可以加载模型 我查看了您发布的源代码 发现有设置模型的参数 然后再加载的模型 请我我该怎么设置这个模型的参数?

**Comment by all150-github** [comment]
def construct_model(model_path):
    input_tensor = Input((None, 40, 3))
    x = input_tensor
    base_conv = 32

    for i in range(3):
        x = Conv2D(base_conv * (2 ** (i)), (3, 3),padding="same")(x)
        x = BatchNormalization()(x)
        x = Activation('relu')(x)
        x = MaxPooling2D(pool_size=(2, 2))(x)
    x = Conv2D(256, (5, 5))(x)
    x = BatchNormalization()(x)
    x = Activation('relu')(x)
    x = Conv2D(1024, (1, 1))(x)
    x = BatchNormalization()(x)
    x = Activation('relu')(x)
    x = Conv2D(len(e2e.chars)+1, (1, 1))(x)
    x = Activation('softmax')(x)
    base_model = Model(inputs=input_tensor, outputs=x)
    base_model.load_weights(model_path)
    return base_model

请问 我该怎么修改这段代码 让它成功加载 ocr_plate_all_gru.h5 这个模型

**Comment by all150-github** [comment]
非常抱歉 向您提出这个问题 因为我对keras不太熟悉
